custom-plugins/msupply-horizontal-bar/package.json

```diff
@@ -3,7 +3,8 @@
   "version": "1.0.2",
   "description": "mSupply Horizontal Bar",
   "scripts": {
-    "build": "grafana-toolkit plugin:build",
+    "copy": "rsync -rP --exclude=node_modules ./ ~/GitHub/msupply-dashboard/custom-plugins/msupply-horizontal-bar/",
```

Comment:
```suggestion
    "copy": "rsync -rP --exclude=node_modules ./ ~/GitHub/msupply-dashboard/custom-plugins/msupply-horizontal-bar/",
    "get": "rsync -rP --exclude=node_modules ~/Github/msupply-dashboard/custom-plugins/msupply-horizontal-bar/ ./",
```

Reply:
@CarlosNZ great suggestions, thanks! 
626243d
The other per-user configuration option is within vscode, it's easy enough to have per user environment settings in vscode.

Adding this into `.vscode/tasks.json` works

```
"tasks": [
        {
            "label": "Copy to repo",
            "type": "shell",
            "osx": {
                "command": "rsync -rP --exclude=node_modules ${workspaceFolder} ~/GitHub/msupply-dashboard/custom-plugins/msupply-horizontal-bar/"
            }
        },
        {
            "label": "Get from repo",
            "type": "shell",
            "osx": {
                "command": "rsync -rP --exclude=node_modules ~/Github/msupply-dashboard/custom-plugins/msupply-horizontal-bar/ ${workspaceFolder}"
            }
        }
```

and it is possible to create your own 'user' tasks in vscode which allows customising. I find the `yarn copy` easier to run, so I'm pushing that rather than tasks.